scp/scpOperand.cpp

```diff
@@ -277,13 +279,17 @@ void SCPOperand::resolveModifiers()
       element_type = element_type | ScType::Node;
       continue;
     }
-    if (modifier == Keynodes::rrel_link)
+    if (modifier == Keynodes::rrel_node_link || modifier == Keynodes::rrel_link)
```

Comment:
you should replace old roles with new roles in ims as well

Reply:
I'll do it later

Reply:
This will be a separate PR

Reply:
can you create issue or something like that so we won't forget?

Reply:
https://github.com/ostis-ai/ims.ostis.kb/pull/44